server/src/infra/1707000751533-AddVectorsToSearchPath.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddVectorsToSearchPath1707000751533 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER DATABASE immich SET search_path TO "$user", public, vectors`);
```

Comment:
We allow a custom database name (see https://immich.app/docs/install/environment-variables#database) so we should probably use that env variable here as well?

Reply:
I believe there are other migrations that have the database name hardcoded to `immich` by the way.

Reply:
> I believe there are other migrations that have the database name hardcoded to `immich` by the way.

I don't think so, using `immich_dev` and haven't any problem yet. Haven't found any occurrence of `immich` db within migrations.